tests/security/selinux/selinux_setup.pm

```diff
@@ -0,0 +1,55 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Setup environment for selinux tests
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#40358
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils;
+use version_utils qw(is_sle is_leap);
+
+sub run {
+    select_console "root-console";
+
+    # for opensuse, e.g, Tumbleweed, add the repo in case rather than by default
+    if (!is_sle && !is_leap) {
+        zypper_call("ar http://download.opensuse.org/tumbleweed/repo/oss/ tumbleweed-Oss");
+    }
+
+    # program 'sestatus' can be found in policycoreutils pkgs
+    zypper_call("in policycoreutils policycoreutils-python");
+
+    my $pkgs = script_output("echo `zypper --no-refresh se selinux | grep -i selinux | grep -v -w srcpackage | cut -d '|' -f 2`");
```

Comment:
still use '--no-refresh' here?

Reply:
Sorry, forgot to remove this one.
Done.